src/devices/Ads1115/AddressSetting.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Ads1115
+{
+    /// <summary>
+    /// ADS1115 I2C Address Setting
+    /// </summary>
+    public enum AddressSetting
+    {
+        /// <summary>
+        /// ADDR Pin connect to GND
+        /// </summary>
+        GND = 0x48,
```

Comment:
I'm not sure I understand what these are. Shouldn't GND and VCC just be connected to ground and power? and SDA and SCL to data and clock pins? What do these addresses represent?

Reply:
@joperezr ADS1115 has a ADDR pin, and ADDR connect to VCC, GND, SDA, SCL represent different I2C address.